LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Afshin Arani
+Copyright (c) 2021-2023 Afshin Arani
```

Comment:
At first you were the only contributor on your spare time, but after that NodeEffect sponsored you to work on this, so you should add NodeEffect here too